src/components/ModalToolbar.css

```diff
@@ -0,0 +1,232 @@
+.timeDisplay {
+    text-align: center;
+    position: fixed;
+    left: 0%;
+    right: 0%;
+    top: 25%;
+    font-size: x-large;
+}
+
+.pageName {
+    text-align: center;
+    position: fixed;
+    right: 0%;
+    margin-right: 1%;
+    font-size: large;
+    top: 32%;
+    margin-right: 3%;
+}
+
+.exitButton {
+    text-align: left;
+    position: fixed;
+    left: 0%;
+    margin-left: 1%;
+    top: 12%;
+    --background-hover: #ffffff;
+    --background-hover-opacity: 5%;
+    --ripple-color: #ffffff;
+}
+
+:root {
+    --ion-color-music: #8a5893;
+    --ion-color-car: #099648;
+    --ion-color-climate: #db973f;
+    --ion-color-navigation: #588193;
+    --ion-color-settings: #565656;
+    --ion-color-blank: #1d1d1d;
+
+    --ion-color-musictext: #dcc6df;
+    --ion-color-cartext: #c7e4d3;
+    --ion-color-climatetext: #f3e5d6;
+    --ion-color-navtext: #cbe4ee;
+    --ion-color-settingstext: #d4d4d4;
+    --ion-color-blanktext: #b3b3b3;
+}
+
+root {
+    --ion-color-music: #8a5893;
```

Comment:
Good job on the CSS vars, this seems like a clean implementation. I actually got Sass working on ARM so it might be better to that SCSS functionality merged and then go for variables. I think it would help max everything cleaner.